Extensions/IISWebAppDeploy/Src/vss-extension.json

```diff
@@ -2,7 +2,7 @@
   "manifestVersion": 1,
   "extensionId": "iiswebapp",
   "name": "IIS Web App Deployment Using WinRM",
-  "version": "1.6.8",
+  "version": "1.235.0",
```

Comment:
Actually, I don't see much need to bump the version to the current sprint.
It might be useful in the tasks repository because there is active work on each task and because this approach has been used for a long time.
But here it looks inconsistent and confusing for customers: Why was the version bumped from 6 to 235.

So, I'd suggest just going with the normal increase of the minor versions.

Reply:
Done